### PR TITLE
UIEH-690: Add Notes accordion to Provider and Resource view pages

### PR DESCRIPTION
**Pull request by Yurii-Danylenko**
https://issues.folio.org/browse/UIEH-690

## Purpose
Integrate NotesSmartAccordion into Provider and Resource show pages

### CODE REVIEW

File: src/components/provider/show/provider-show.js [hunk 1]
```diff
@@ -258,7 +260,17 @@ class ProviderShow extends Component {
               {hasProxy && this.renderProxy()}
               {hasToken && this.renderToken()}
             </Accordion>
-          )}
+          )
+        }
+
```
Comment (MaksymDryha) [inline src/components/provider/show/provider-show.js]: please remove empty line

File: src/components/resource/resource-show.js [hunk 2]
```diff
@@ -553,6 +555,15 @@ class ResourceShow extends Component {
                 isOpen={sections.resourceShowAgreements}
                 onToggle={this.handleSectionToggle}
               />
+
```
Comment (MaksymDryha) [inline src/components/resource/resource-show.js]: please remove empty line

File: src/components/resource/resource-show.js [hunk 3]
```diff
@@ -553,6 +555,15 @@ class ResourceShow extends Component {
...
+              <NotesSmartAccordion
+                domainName="eholdings"
```
Comment (Godlevskyi) [inline src/components/resource/resource-show.js]: Please use constant for domain and entity type

File: src/components/provider/show/provider-show.js [hunk 4]
```diff
@@ -258,7 +260,17 @@ class ProviderShow extends Component {
...
+        <NotesSmartAccordion
+          domainName="eholdings"
```
Comment (Godlevskyi) [inline src/components/provider/show/provider-show.js]: Please use constants for domain and entity type

**Pull request merged**